pymyq/api.py

```diff
@@ -1,293 +1,214 @@
 """Define the MyQ API."""
 import asyncio
-import logging
 from datetime import datetime, timedelta
-from typing import Optional
+import logging
+from typing import Dict, Optional
 
 from aiohttp import ClientSession
 from aiohttp.client_exceptions import ClientError
 
-from .errors import MyQError, RequestError, UnsupportedBrandError
+from .device import MyQDevice
+from .errors import (
+    InvalidCredentialsError,
+    MyQError,
+    RequestError,
+    UnsupportedBrandError,
+)
 
 _LOGGER = logging.getLogger(__name__)
 
-API_BASE = 'https://myqexternal.myqdevice.com'
-LOGIN_ENDPOINT = "api/v4/User/Validate"
-DEVICE_LIST_ENDPOINT = "api/v4/UserDeviceDetails/Get"
-
-DEFAULT_TIMEOUT = 1
-DEFAULT_REQUEST_RETRIES = 3
-
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
+API_VERSION = 5
+API_BASE = "https://api.myqdevice.com/api/v{0}".format(API_VERSION)
 
+DEFAULT_REQUEST_RETRIES = 5
+DEFAULT_STATE_UPDATE_INTERVAL = timedelta(seconds=5)
 DEFAULT_USER_AGENT = "Chamberlain/3.73"
 
+NON_COVER_DEVICE_FAMILIES = "gateway"
+
 BRAND_MAPPINGS = {
```

Comment:
I believe these have all been rolled into a single AppID. Try just using JVM/G9Nwih5BwKgNCjLxiFUQxQijAebyyg8QUHr7JOrP+tuPb8iHfRHKwTmDzHOu

Reply:
bb1d5a4 should cover this.